### PR TITLE
Space dolphins

### PR DESCRIPTION
**Pull request by Time-Green**

### CODE REVIEW

File: code/modules/food_and_drinks/food/snacks_other.dm [hunk 1]
```diff
@@ -315,4 +315,28 @@
 	desc = "The former skull of a damned human, filled with goliath meat. It has a decorative lava pool made of ketchup and hotsauce."
 	icon_state = "stuffed_legion"
 	bonus_reagents = list("vitamin" = 3, "capsaicin" = 1, "tricordrazine" = 5)
-	list_reagents = list("nutriment" = 5, "vitamin" = 5, "capsaicin" = 2, "tricordrazine" = 10,)
\ No newline at end of file
+	list_reagents = list("nutriment" = 5, "vitamin" = 5, "capsaicin" = 2, "tricordrazine" = 10,)
+
+/obj/item/weapon/reagent_containers/food/snacks/dolphincereal
+	name = "dolphin cereal"
+	desc = "Finest dolphin skin flakes. This looks flippin' disguisting."
```
Comment (ShadowDeath6) [inline code/modules/food_and_drinks/food/snacks_other.dm]: Disguisting

**Comment by george99g** [comment]
I wrote the original code for a bit of this.
Can confirm, not a full port.
Missing these files:
https://github.com/yogstation13/yogstation/blob/a2a12d87d0000ab5d0ddb0ea3ba4fea2f62f6ec4/code/modules/food%26drinks/recipes/tablecraft/recipes_misc.dm
https://github.com/yogstation13/yogstation/blob/a2a12d87d0000ab5d0ddb0ea3ba4fea2f62f6ec4/code/modules/food%26drinks/recipes/tablecraft/recipes_soup.dm

**Comment by Time-Green** [comment]
I just realised what @ShadowDeath6 meant

**Comment by george99g** [comment]
I didn't even realize I typed that in until now...

I blame BYOND's shitty IDE not giving me a spell-checker.

Also, squash your damn commits already!

**Comment by Time-Green** [comment]
Squash?

**Comment by oisin100** [comment]
just checking, How did you port the DMI files? Copypasta or using the inbuilt Export/Import in DreamMaker?

**Comment by Time-Green** [comment]
I used export and import.
schermopname 247

**Comment by oisin100** [comment]
Coolio, Just making sure.

**Pull request merged**